src/main/java/at/aau/serg/websocketdemoserver/websocket/broker/WebSocketBrokerController.java

```diff
@@ -74,10 +74,23 @@ public void playCard(CardPlayRequest playCardRequest) throws Exception {
         cardPlayedRequest.setCardType(card.getCardType());
         cardPlayedRequest.setColor(card.getColor());
         cardPlayedRequest.setValue(String.valueOf(card.getValue()));
+        endTurnForActivePlayer(playCardRequest.getLobbyCode());
+
+        Lobby currentLobby = lobbyManager.getLobbyByID(playCardRequest.getLobbyCode());
+        // if (currentLobby.isCurrentTrickDone()) {
```

Comment:
Bin mir nicht sicher ob das automatische Mergen hier funktionieren wird -> am besten denn Main branch händisch in deinen Mergen um zu schauen ob hier meine Änderungen richtig angewendet werden

Reply:
Der PlayCard Endpoint wurde im nachfolgenden Commit auf den Stand von  main gebracht